Annonce de sondage pour des ateliers de majeures 2022

gtedavid opened the pull request:
Marc, j'aurais besoin de ton github pls

@melisandeGB Hésite pas à corriger, proposer des améliorations ou à relire.
@noftaly idem ;)

Priority : Medium to High

gtedavid committed:
Create ANNOUNCMENT_POLL_ECHANGE_MAJEURES_JUIN_2022.md

Marc, j'aurais besoin de ton github pls

@melisandeGB Hésite pas à corriger, proposer des améliorations ou à relire.
@noftaly idem ;)

Priority : Medium to High

gtedavid committed:
Format + where

Adaptation du format + lieu du déroulement de l'évent

noftaly commented on resources/ANNOUNCMENT_POLL_ECHANGE_MAJEURES_JUIN_2022.md:
```diff
@@ -0,0 +1,21 @@
+Souhaiterez-vous qu'on organise des ateliers avec des étudiants des majeures :mortar_board: ?
+
+
+1️⃣Oui, c'est une bonne idée, ça m'aiderait ou serait intéressant
+
+2️⃣Non, je ne pense pas que cela soit utile
+
+3️⃣Mon choix est déjà fait
+
+4️⃣Ne souhaite pas se prononcer
```
```suggestion
:one: Oui, c'est une bonne idée, ça m'aiderait ou serait intéressant

:two: Non, je ne pense pas que cela soit utile

:three: Mon choix est déjà fait

:four: Ne souhaite pas se prononcer
```

noftaly reviewed (approved):
C'est bon pour moi, juste modifier les emojis par précaution.

Après à titre perso je trouve qu'il y a beaucoup trop de retours à la ligne, ca fait trop aéré et déstructuré 😅 

Et oublie pas que 2 lignes vides = nouveau message 😅

noftaly commented on resources/ANNOUNCMENT_POLL_ECHANGE_MAJEURES_JUIN_2022.md:
```diff
@@ -0,0 +1,21 @@
+
+Ceci est un sondage pour vous.
+
+<@370940255026675715> et moi (David) avons envie d'organiser sur 📖Ef'Réussite⭐ un évent ou des petits ateliers pour échanger :speech_balloon: :speech_left: sur les majeures !
```
```suggestion
<@370940255026675715> et moi (David) avons envie d'organiser sur 📖Ef'Réussite⭐ un évent ou des petits ateliers pour échanger :speech_balloon: :speech_left: sur les majeures !
```
btw "et moi (David)" ca va rendre très bizarre comme c'est le bot qui l'envoie 😅 soit tu te mentionnes comme si c'était une troisième personne, soit tu dis juste "Nous"

gtedavid replied:
true 😅
Mais au moins tu sais qui gérerait l'événement 😅

gtedavid committed:
Ajustement du fomat

Ajustement du format pour les choix

Co-authored-by: Elliot <elliot.maisl@gmail.com>

noftaly commented:
(oublie pas d'enlever tous les retours à la lignes pour pas que ca créé 30 messages 😅 )

gtedavid commented:
Ok pour les lignes
C'est juste que j'avais pas l'impression que c'était assez espacé 😅

gtedavid committed:
Correction des lignes

Est-ce que c'est mieux @noftaly ?
Je me demande s'il faut pas enlever les lignes entre les 1 2  3 4

gtedavid committed:
Correction de Marc et "moi"

Nous à la place des orga

melisandeGB commented on resources/ANNOUNCMENT_POLL_ECHANGE_MAJEURES_JUIN_2022.md:
```diff
@@ -0,0 +1,18 @@
+Nous avons envie d'organiser sur 📖Ef'Réussite⭐ un évent ou des petits ateliers pour échanger :speech_balloon: :speech_left: sur les majeures !
+
+A savoir, qu'on verra ce qu'on pourrait faire par rapport à la nouvelle majeure 🔒"*Cybersécurité & Cloud*"☁️.
+Mais ça n'est pas là le sujet de discussion pour le moment....
```
```suggestion
Mais ce n'est pas le sujet de discussion pour le moment....
```

melisandeGB commented on resources/ANNOUNCMENT_POLL_ECHANGE_MAJEURES_JUIN_2022.md:
```diff
@@ -0,0 +1,18 @@
+
+Nous avons envie d'organiser sur 📖Ef'Réussite⭐ un évent ou des petits ateliers pour échanger :speech_balloon: :speech_left: sur les majeures !
+
+A savoir, qu'on verra ce qu'on pourrait faire par rapport à la nouvelle majeure 🔒"*Cybersécurité & Cloud*"☁️.
```
```suggestion
On verra ce qu'on pourrait faire par rapport à la nouvelle majeure 🔒"*Cybersécurité & Cloud*"☁️.
```

gtedavid committed:
Correction "ça"-> "ce" et "là"->"le"

Co-authored-by: Mélisande  <72623143+melisandeGB@users.noreply.github.com>

gtedavid committed:
Simplification

Suppression du "A savoir que" (rend la phrase plus lourd)

Co-authored-by: Mélisande  <72623143+melisandeGB@users.noreply.github.com>
